contestant_containers/arch/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM docker.io/blackarchlinux/blackarch:latest
+LABEL maintainer "Trevor Bryant <@apporima>"
+
+# Temporary patch to build on Ubuntu 20.04/10 & GitHub CI
+# https://bugs.archlinux.org/index.php?do=details&task_id=69563
+RUN patched_glibc=glibc-linux4-2.33-4-x86_64.pkg.tar.zst && \
+    curl -LO "https://repo.archlinuxcn.org/x86_64/$patched_glibc" && \
+    bsdtar -C / -xvf "$patched_glibc"
+
+# Install packages
+RUN pacman -Syu --noconfirm git make gcc pkgconf which python-pip wget openssh vim tmux screen supervisor iw mlocate man pciutils less bash-completion xorg-server-xvfb x11vnc dialog xfce4 xfce4-goodies xfce4-power-manager inetutils \
```

Comment:
BlackArch maintainer says to add blackarch-config-xfce and blackarch-menus

Reply:
Thanks. Those are obviously much needed and way better. I'm surprised I didn't find them in the [tools](https://blackarch.org/tools.html) repository, but it is what it is.